### PR TITLE
Bump Azure.ResourceManager.Maps version to 1.1.0

### PR DESCRIPTION
**Pull request by dubiety**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by dubiety** [comment]
Hi @archerzz @ArcturusZhang @ArthurMa1978, I'm planning to release `Azure.ResouceManager.Maps` to `1.1.0` (non-beta version), can you help to approve? We need to use the functionality in `1.1.0-bete.x` on production, so we need a formal version. Thank you very much.

File: sdk/maps/Azure.ResourceManager.Maps/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0-beta.4 (Unreleased)
+## 1.1.0 (Unreleased)
```
Comment (live1206) [inline sdk/maps/Azure.ResourceManager.Maps/CHANGELOG.md:3]: Please specify the release date for this new release

Comment (dubiety) [reply sdk/maps/Azure.ResourceManager.Maps/CHANGELOG.md]: Done. Thanks for the suggestion.

**Review by live1206 — APPROVED** [review]

**Pull request merged**